app/Services/JsonService.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+if (!function_exists('array_is_list')) {
```

Comment:
suggestion: Add declare(strict_types=1); in new class and files

Reply:
At the moment, I do not believe there is much value in doing so. We can catch errors during development and test phase with e.g. PHPStan instead of risking some crashes in production. I suggest we move to that when we are higher up with our PHPStan levels.

Reply:
when you add a new class, it is wise to add this at the beginning of the file so that php does not transform the types in an uncontrollable way.